server/scripts/seeds.ts

```diff
@@ -0,0 +1,15 @@
+import { NestFactory } from '@nestjs/core';
+import { AppModule } from '../src/app.module';
+import { SeedsService } from '@services/seeds.service';
+
+async function bootstrap() {
```

Comment:
Loading the seed service in the application context 
https://docs.nestjs.com/standalone-applications